Open-Source/README.md

```diff
@@ -0,0 +1,48 @@
+# Open Source
+
+## Introduction
```

Comment:
Remove the Introduction line, and make `What is Open Source` as a h2(two hashtags`